ios/FluentUI.Demo/FluentUI.Demo/Demos/ActivityIndicatorDemoController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class ActivityIndicatorDemoController: DemoController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let hidesWhenStoppedSettingView = createLabelAndSwitchRow(labelText: "Hides when stopped",
```

Comment:
nit: can we stop using this pattern because it doesn't look good in larger text?